tgui/packages/tgui/styles/interfaces/IntegratedCircuit.scss

```
@@ -3,27 +3,283 @@
+    rgba(7, 11, 18, 1) 55%,
+    rgba(10, 14, 22, 1) 100%
+  );
+}
+
+// Весь фрейм окна (включая TitleBar с полем имени), пока включён LGBTK+ режим.
+.Layout.Window:has([data-ic-rainbow]) {
+  animation: IntegratedCircuit__rainbowHue 4.5s linear infinite;
+  will-change: filter;
+}
+
+@keyframes IntegratedCircuit__rainbowHue {
```

Stylelint (17.4.0)

[error] Expected keyframe name "IntegratedCircuit__rainbowHue" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)